src/index.js

```diff
@@ -4,9 +4,11 @@ import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
+import LoginPage from './pages/LoginPage';
+
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <LoginPage />
```

Comment:
I know this is the way to test the component without the base architecture, but we shouldn't merge this change in the App.js

Reply:
Yes, I know, that particular commit wasn't really intended, besides of the other files commit, we should merge the develop branch with the react router and initial setup we made and start our branches from that in order to avoid these mistakes, to take in count for our meetings tomorrow.